docs/products/kafka/howto/kcat.rst

```diff
@@ -0,0 +1,112 @@
+Use kcat with Aiven for Apache Kafka
+====================================
+
+`kcat <https://github.com/edenhill/kcat>`_ (formerly known as kafkacat) is a generic non-JVM producer and consumer for Apache Kafka. 
+
+It can be used to produce and consume records to Apache Kafka topics as well as to list service configurations.
```

Comment:
maybe put it next to previous sentence? they seem to belong to one paragraph

Reply:
done, correct! Thanks!